dags/hdx_adm_dags.py

```diff
@@ -18,8 +18,15 @@ def create_hdx_adm0_dag(country, schedule_interval, catchup, config, default_arg
     )
     adm0_transform = HDXAdm0Operator(
         task_id=f"hdx_adm0_{country}_transform",
-        country=country,
-        config=config,
+        source='cod',
+        input_filename="/opt/data/test/yem_adm_govyem_cso_ochayemen_20191002_GPKG.zip",
```

Comment:
These will come from the config later as well, right?

Reply:
Yes correct